src/components/Layouts/Primary/TopBar/Items/SearchMenu/SearchDialog.tsx

```diff
@@ -338,30 +196,30 @@ export const SearchDialog: React.FC<SearchDialogProps> = ({ handleClose }) => {
           const filteredOptions = options.filter((option) =>
             option.id
               ? true
-              : option.name
+              : option.title
                   .toLowerCase()
                   .includes(wildcardSearch.toLowerCase()),
           );
 
           if (contacts && contacts.totalCount > contacts.nodes.length) {
             filteredOptions.splice(contacts.nodes.length, 0, {
-              name: t(
+              title: t(
                 `And ${contacts.totalCount - contacts.nodes.length} more`,
```

Comment:
Not your code, but can we make this a static string?

```suggestion
                'And {{count}} more',
```